packages/aws-cdk-lib/aws-cloudwatch/README.md

```diff
@@ -387,6 +387,25 @@ only supports filtering by `unit` for Alarms, not in Dashboard graphs.
 Please see the following GitHub issue for a discussion on real unit
 calculations in CDK: https://github.com/aws/aws-cdk/issues/5595
 
+## Anomaly Detection Alarms
+
+To create an alarm based on anomaly detection, you can use the `createAnomalyDetectionAlarm` method on a `Metric` or `MathExpression` object:
```

Comment:
I'm just curious why do you decide to create a function here instead of a construct `new cloudwatch.AnomalyDetectionAlarm(...)`

Reply:
Main motivations:
- Less refactoring.
- Less risk of breaking the existing `Alarm` construct.
- The handling of the anomaly detection case require limited amount of changes in the existing `Alarm` construct.

Tradeoff:
- It is still possible to not use the new helper methods and give a wrong metric to the existing `Alarm` construct.